fastai/callback/wandb.py

```diff
@@ -120,7 +120,7 @@ def after_fit(self):
                 log_model(self.save_model.last_saved_path, metadata=metadata)
         self.run = True
         if self.log_preds: self.remove_cb(FetchPredsCallback)
-        wandb.log({}) # ensure sync of last step
+        wandb.log({}, step=self._wandb_step) # ensure sync of last step
```

Comment:
I believe this would be equivalent to just removing that line.

Reply:
Maybe the solution should be to do:
```
# ensure sync of last step
wandb.log({})
self._wandb_step += 1
```

Reply:
@borisdayma can you run all three variants in your test suite?

Reply:
@borisdayma Looks like a reset to upstream, closes an open PR. In any case: I just tested the your last suggestion and this works!